From stern@rowland.harvard.edu  Fri Aug 10 11:08:05 2007
From: Alan Stern <stern@rowland.harvard.edu>
Date: Wed, 8 Aug 2007 17:16:12 -0400 (EDT)
Subject: USB: remove DEBUG definition from dummy_hcd
To: Greg KH <greg@kroah.com>
Cc: Gabriel C <nix.or.die@googlemail.com>,  USB development list <linux-usb-devel@lists.sourceforge.net>
Message-ID: <Pine.LNX.4.44L0.0708081713280.2737-100000@iolanthe.rowland.org>


This patch (as958) removes an unneeded and unwanted #define line from
dummy_hcd.

Signed-off-by: Alan Stern <stern@rowland.harvard.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/dummy_hcd.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/usb/gadget/dummy_hcd.c
+++ b/drivers/usb/gadget/dummy_hcd.c
@@ -34,8 +34,6 @@
  * bypassing some hardware (and driver) issues.  UML could help too.
  */
 
-#define DEBUG
-
 #include <linux/module.h>
 #include <linux/kernel.h>
 #include <linux/delay.h>
